src/elements/content-picker/selectionCellRenderer.js

```diff
@@ -15,11 +15,13 @@ export default (
     extensionsWhitelist: string[],
     hasHitSelectionLimit: boolean,
     isRadio: boolean,
+    selected: Array<BoxItem>,
```

Comment:
Should we be passing the entire selected collection into each row? Can the parent pass in an `isSelected` value, instead?